js/TUI.js

```diff
@@ -12,28 +12,29 @@
  * 
  * TUI stands for Tweak UI.
  *
- * Requires js/util.js and the YUI Dom and Cookie libraries.
+ * Requires js/util.js and jquery
  *
  * See template/en/default/bug/create/create.html.tmpl for a usage example.
  */
 
 var TUI_HIDDEN_CLASS = 'bz_tui_hidden';
 var TUI_COOKIE_NAME  = 'TUI';
```

Comment:
This shouldn't be called TUI_COOKIE_NAME any more, as it doesn't name a cookie. TUI_LOCALSTORAGE_TAG?